hardhat.config.ts

```diff
@@ -15,11 +15,7 @@ import 'tsconfig-paths/register';
 
 const networks: NetworksUserConfig =
   env.isHardhatCompile() || env.isHardhatClean() || env.isTesting()
-    ? {
-        hardhat: {
-          chainId: 1, // We are trying to mock mainnet
-        },
-      }
```

Comment:
I was going to ping you off-github because after talking about this in the other PR, I thought probably best-case was to take the hardcoded addresses. Great to think we were thinking the same thing!